pay/example/ios/Runner/Info.plist

```diff
@@ -43,5 +43,7 @@
 	<false/>
 	<key>CADisableMinimumFrameDurationOnPhone</key>
 	<true/>
+	<key>UIApplicationSupportsIndirectInputEvents</key>
```

Comment:
What's the use for this flag?

Reply:
I didn't add it intentionally, it was added when I ran flutter pub get with the latest version.

You can read about it here: https://developer.apple.com/documentation/bundleresources/information_property_list/uiapplicationsupportsindirectinputevents